### PR TITLE
bindings/capi: Added gradient getter

### PR DESCRIPTION
**Pull request by mmaciola**
added gradient's color_stops and spread getter for Capi binding
add test code in testCapi.c

### CODE REVIEW

File: src/bindings/capi/tvgCapi.cpp [hunk 1]
```diff
@@ -428,12 +428,24 @@ TVG_EXPORT Tvg_Result tvg_gradient_set_color_stops(Tvg_Gradient* grad, const Tvg
     if (!grad) return TVG_RESULT_INVALID_ARGUMENT;
     return (Tvg_Result) reinterpret_cast<Fill*>(grad)->colorStops(reinterpret_cast<const Fill::ColorStop*>(color_stop), cnt);
 }
+TVG_EXPORT Tvg_Result tvg_gradient_get_color_stops(Tvg_Gradient* grad, const Tvg_Color_Stop** color_stop, uint32_t* cnt)
```
Comment (hermet) [inline src/bindings/capi/tvgCapi.cpp:431]: please keep code consistency  we have 2 empty lines between functions.

File: src/bindings/capi/tvgCapi.cpp [hunk 2]
```diff
@@ -428,12 +428,24 @@ TVG_EXPORT Tvg_Result tvg_gradient_set_color_stops(Tvg_Gradient* grad, const Tvg
...
+    *cnt = reinterpret_cast<Fill*>(grad)->colorStops(reinterpret_cast<const Fill::ColorStop**>(color_stop));
+    return TVG_RESULT_SUCCESS;
+}
 
 TVG_EXPORT Tvg_Result tvg_gradient_set_spread(Tvg_Gradient* grad, const Tvg_Stroke_Fill spread)
 {
     if (!grad) return TVG_RESULT_INVALID_ARGUMENT;
     return (Tvg_Result) reinterpret_cast<Fill*>(grad)->spread((FillSpread)spread);
 }
+TVG_EXPORT Tvg_Result tvg_gradient_get_spread(Tvg_Gradient* grad, Tvg_Stroke_Fill* spread)
```
Comment (hermet) [inline src/bindings/capi/tvgCapi.cpp:443]: please keep code consistency  we have 2 empty lines between functions.

**Review by hermet — CHANGES_REQUESTED** [review]
Please check comments.

File: inc/thorvg_capi.h [hunk 3]
```diff
@@ -161,7 +161,9 @@ TVG_EXPORT Tvg_Gradient* tvg_radial_gradient_new();
 TVG_EXPORT Tvg_Result tvg_linear_gradient_set(Tvg_Gradient* grad, float x1, float y1, float x2, float y2);
 TVG_EXPORT Tvg_Result tvg_radial_gradient_set(Tvg_Gradient* grad, float cx, float cy, float radius);
```
Comment (mihashco) [inline inc/thorvg_capi.h:162]: Please add getters for linear_gradient_set and radial_gradient_set in this commit.

**Pull request merged**